emuu-client/src/components/CreatorsPage/UserInfo.js

```diff
@@ -25,7 +24,6 @@ function UserInfo() {
   });
 
   return (
-
     <div className="Main-UserInfo-Container">
       <h3 className="about">About</h3>
 
```

Comment:
Good use of prettier, I see its becoming a more consistent thing all of us are utilizing.